data.sql

```diff
@@ -1,5 +1,7 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Agumon', '02-03-2020', 0, true, 10.23),
+       ('Gabumon', '11-15-2018', 2, true, 8),
+       ('Pikachu', '01-07-2021', 1, false, 15.04),
+       ('Devimon', '05-12-2017', 5, true, 11)
```

Comment:
- [ ] Great job so far 👏🏻 However, you did not follow the PostgreSQL DATE data type format. When storing a date value, PostgreSQL uses the `yyyy-mm-dd` format e.g. 1994-10-27. This format is also used in PostgreSQL for [inserting data](https://www.prisma.io/dataguide/postgresql/inserting-and-modifying-data).